docs/Using Fleet/Automations.md

```diff
@@ -130,9 +130,9 @@ Follow the steps below to configure Jira or Zendesk as a ticket destination:
 
 ## Host status automations
 
-Host status automations send a webhook request if a configured percentage of hosts have not checked in to Fleet for a configured number of days.
+Host status automations send a webhook request if a configured percentage of hosts have not checked in to Fleet for a configured number of days. This can be customized [globally](https://fleetdm.com/docs/using-fleet/configuration-files#organization-settings) or [per-team](https://fleetdm.com/docs/using-fleet/configuration-files#teams).
```

Comment:
```suggestion
Host status automations send a webhook request if a configured percentage of hosts have not checked in to Fleet for a configured number of days. This can be customized [globally](https://fleetdm.com/docs/configuration/configuration-files#organization-settingss) or [per-team](https://fleetdm.com/docs/configuration/configuration-files#teams).
```